src/monitor.tf

```diff
@@ -153,6 +153,8 @@ module "web_test_api" {
   request_url                       = format("https://%s%s", each.value.host, each.value.path)
   ssl_cert_remaining_lifetime_check = 7
   expected_http_status              = each.value.expected_http_status
+  content_validation                = "null"
+  auto_mitigate                     = true
```

Comment:
```suggestion
```
auto_mitigate = false is the main reason introduced in v2.5.0. We want a continuous alert when some endpoints and certs have a problem because the team must look or fix the issue.

in v2.8.2 we fixed content_validation default value so you can remove it

Reply:
OK, wasn't clear to me the auto_mitigate issue. I'll put it back.

Reply:
Regarding the upgrade to v2.8.2 the reason I haven't done it yet is the following:

```
│ Error: Unsupported Terraform Core version
│
│   on .terraform/modules/web_test_api/application_insights_web_test_preview/versions.tf line 2, in terraform:
│    2:   required_version = ">= 1.1.0"
```

which collides with:

```
cat ../.terraform-version
1.0.7
```

How should we proceed?

Reply:
you can upgrade terraform to latest version (1.1.7) safetly :)